### ISSUE TITLE
Redesign GUI client-server model

### ISSUE THREAD

**Issue by aalexandrov** [stratosphere/stratosphere#297]
Factor out job manager status information as REST service running inside the same process. Implement visualization server as a separate web application that runs on the client-side and renders data fetched from from the job manager RESTful API.

**Comment by markus-h** [comment]
Currently we have a jetty server running inside the jobmanger process that is providing JSON data for job information. The visualization is done by the browser using HTML5 and JavaScript. So the only change should be to make the API RESTful?

**Comment by aalexandrov** [comment]
Yes, probably just rethink the API in sight of what we want to show in the browser. Is the HTML5 and the JavaScript content served by the embedded Jetty server as well or is this done by somebody else?

**Comment by markus-h** [comment]
Yes it is served by the Jetty as well
